main.py

```diff
@@ -55,6 +56,24 @@ def shutdown(self):
         """Do nothing."""
         log.info('NApp kytos/topology shutting down.')
 
+    @staticmethod
+    def _get_metadata():
+        """Return a JSON with metadata."""
+        try:
+            metadata = request.get_json()
+            content_type = request.content_type
+        except BadRequest:
+            result = 'The request body is not a well-formed JSON.'
+            raise BadRequest(result)
+        if content_type is None:
+            result = ('The request body is empty.')
+            raise BadRequest(result)
+        if metadata is None:
+            result = ('The content type must be application/json and not '
+                      f'{content_type}.')
```

Comment:
Same problem as before: you're checking for 'empty metadata' e telling the user the problem is the 'content-type'. 

```suggestion
            if content_type != 'application/json':
                result = ('The content type must be application/json '
                          f'(received {content_type}).')
            else:
                result = 'Metadata is empty.'
```